core/src/components/toast/toast.scss

```
@@ -141,7 +150,7 @@
   z-index: 0;
 }
 
-.toast-icon {
+.toast-button-icon {
```

```suggestion
.toast-icon,
.toast-button-icon {
```

These styles should also apply to `.toast-icon` here so we do not need to duplicate it above.